File: 2350-find-closest-number-to-zero/2350-find-closest-number-to-zero.cpp
class Solution {
public:
    int findClosestNumber(vector<int>& nums) {
        int max_value[2];
        int value = numeric_limits<int>::max();
        int res = 0;
       for(int i: nums){
            int temp = (i - 0 > 0) ? i : -i;
            if(temp < value){
                value = (i - 0 > 0) ? i : -i;
                res = i;
            }
            if(temp == value){
                res = (i > res) ? i : res;
            }

       } 
       return res;
    }
};

File: 2350-find-closest-number-to-zero/2350-find-closest-number-to-zero.py
class Solution:
    def findClosestNumber(self, nums: List[int]) -> int:
        res = nums[0]
        for i in nums:
            if abs(i) == abs(res):
                res = max(i,res)
            elif abs(i) < abs(res):
                res = i
        return  res